app/graphql/mutations.js

```diff
@@ -99,3 +99,95 @@ export const UPDATE_EQUIPMENT_FIRMWARE = gql`
     }
   }
 `;
+
+export const DELETE_TRACK_ASSIGNMENT = gql`
+  mutation UpdateEquipmentFirmware($firmwareTrackId: ID!, $firmwareVersionId: ID!) {
+    deleteFirmwareTrackAssignment(
+      firmwareTrackId: $firmwareTrackId
+      firmwareVersionId: $firmwareVersionId
+    ) {
+      trackRecordId
+      firmwareVersionRecordId
+      modelId
+      createdTimestamp
+      lastModifiedTimestamp
+    }
+  }
+`;
+
+export const DELETE_FIRMWARE = gql`
+  mutation DeleteFirmware($id: ID!) {
+    deleteFirmware(id: $id) {
+      id
+    }
+  }
+`;
```

Comment:
move Delete to the bottom, things should be listed in the order of CRUD for ease of readability